workflow_imaging/ingest.py

```diff
@@ -1,66 +1,77 @@
 import scanreader
+import pathlib
+import csv
 
-from workflow_imaging.pipeline import subject, imaging, scan, Session, Equipment
-from workflow_imaging.paths import get_imaging_root_data_dir
+from .pipeline import subject, imaging, scan, Session, Equipment
+from .paths import get_imaging_root_data_dir
 
 from elements_imaging.readers import get_scanimage_acq_time, parse_scanimage_header
 
 
-def ingest():
-    # ---------- Insert new "Session" and "Scan" ----------
-    data_dir = get_imaging_root_data_dir()
+def ingest_subjects():
+    # -------------- Insert new "Subject" --------------
+    with open('./user_data/subjects.csv', newline='') as f:
+        subjects_dict = list(csv.DictReader(f, delimiter=','))
+
+    print(f'\n---- Insert {len(subjects_dict)} entry(s) into subject.Subject ----')
+    subject.Subject.insert(subjects_dict, skip_duplicates=True)
+
+
+def ingest_sessions():
+    root_data_dir = get_imaging_root_data_dir()
+
+    # ---------- Insert new "Session" and "Scan" ---------
+    with open('./user_data/sessions.csv', newline='') as f:
+        sessions_dict = list(csv.DictReader(f, delimiter=','))
```

Comment:
I would just call this `sessions`.  The variable name should describe the object it represents and not its datatype.

Reply:
thanks @dimitri-yatsenko , changed to `input_sessions` to indicate that this is a user-input information (read from csv)